debian/linglong-bin.postinst

```diff
@@ -1,98 +1,101 @@
-#!/bin/sh
+#!/bin/bash
 
 # SPDX-FileCopyrightText: 2023 UnionTech Software Technology Co., Ltd.
-#
```

Comment:
All other scripts we written in this project has almost the same copyright comment. So let's just keep that empty line.